crates/holochain/src/core/workflow/app_validation_workflow/tests.rs

```diff
@@ -408,6 +487,33 @@ async fn commit_invalid(
     (invalid_header_hash, entry_hash)
 }
 
+// Need to "hack holochain" because otherwise the invalid
+// commit is caught by the call zome workflow
+async fn commit_invalid_post(
+    bob_cell_id: &CellId,
+    handle: &ConductorHandle,
+    dna_file: &DnaFile,
+) -> (HeaderHash, EntryHash) {
+    // Bananas are not allowed
+    let entry = Post("Banana".into());
+    let entry_hash = EntryHash::with_data_sync(&Entry::try_from(entry.clone()).unwrap());
+    // Create call data for the 3rd zome Create
+    let (bob_env, call_data) = CallData::create_for_zome(bob_cell_id, handle, dna_file, 2).await;
+    // 9
```

Comment:
What does 9 refer to here?

Reply:
I have been putting header sequence numbers above the manual commits in tests so it's easier to debug